addons/mail/static/src/views/web/fields/html_composer_message_field/content_expandable_plugin.js

```diff
@@ -0,0 +1,49 @@
+import { Plugin } from "@html_editor/plugin";
+import { parseHTML } from "@html_editor/utils/html";
+
+export class ContentExpandablePlugin extends Plugin {
+    static id = "contentexpandable";
+    static dependencies = ["dom", "history"];
+    resources = {
+        start_edition_handlers: this.insertReplyContent.bind(this),
+        clean_for_save_handlers: ({ root }) => this.cleanForSave(root),
+    };
+
+    insertReplyContent() {
+        const mailQuoteElement = this.editable.querySelectorAll('*[data-o-mail-quote="1"]');
+        for (const element of mailQuoteElement) {
+            element.removeAttribute("data-o-mail-quote");
+        }
+        const paragraph = parseHTML(
+            this.document,
+            `<p><br/><button id="viewMore" contenteditable="false" class="badge rounded-pill border-0 m-1 py-0 px-1"><i class="fa fa-ellipsis-h fa-lg"/></button></p>`
+        );
+        const button = paragraph.querySelector("#viewMore");
+        this.addDomListener(button, "click", () => this.onClickViewButton());
```

Comment:
You could also give it the bounded function instead of a brand new arrow function.